test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -41,4 +43,5 @@ dependencies {
   api "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:${versions.jackson}"
   api "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
   api 'net.minidev:json-smart:2.4.8'
+  api 'org.apache.zookeeper:zookeeper:3.8.0'
```

Comment:
@ryanbogan  https://github.com/opensearch-project/OpenSearch/pull/6010 ?

Reply:
Yeah everything with zookeeper is from that PR - I'll rerun gradle check once that one is merged